index.php

```diff
@@ -0,0 +1,26 @@
+<?php
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <title>Capybara Canvas</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="normalize.css" />
+    <link rel="stylesheet" href="style.css" />
+    <script src='paintBrush.js' defer></script>
+</head>
+<body>
+<div>
+    <div class="toolbar">
+        <button class="painter"></button>
```

Comment:
whats the point in these buttons? they don't do anything

Reply:
These are the buttons from story 2, we can comment these out if needed.